server/app.ts

```diff
@@ -37,6 +39,7 @@ export default function createApp(services: Services): express.Application {
   app.use(authorisationMiddleware())
   app.use(setUpCsrf())
   app.use(setUpCurrentUser(services))
+  app.use(setUpFrontendComponents(dataAccess))
```

Comment:
Hmm, I'm not sure how comfortable I am with this (injecting all of the data access clients)
I think this is why when I subtasked it, one of the subtasks was to create a service
I think I'd rather have a front end components service that that has the relevant data access client class injected into it (ie. the same pattern as what we already have)

Could you have a look at creating a service class please, that has one method in it (`getFooter`) that calls the client class and method
Then when we do the header we'll add another method to the service called `getHeader` (but dont do that in this PR)

Reply:
Removed as part of rewrite